packages/frontend/src/constants.ts

```diff
@@ -44,3 +44,23 @@ export const EMPTY_USER = {
   role: UserRole.Coordinator,
   status: UserStatus.Inactive,
 };
+
+export const DEVELOPMENT_ENVIRONMENT_URL = 'http://localhost:5173/';
```

Comment:
Maybe change it to array ['localhost', 'epmgcip-uva-develop'] and check DEVELOPMENT_ENVIRONMENT_URL .some(url => currentUrl.includes(url)) ? Because if buttons are hidden everywhere expect localhost, we have no possibility to login on tmp envs like https://epmgcip-uva-develop--pr15-feature-epmgcipuva-6-8mngx7pu.web.app/

Reply:
Done